pkg/util/kubernetes/apiserver/util.go

```diff
@@ -70,6 +70,11 @@ type syncInformerResult struct {
 func SyncInformersReturnErrors(informers map[InformerName]cache.SharedInformer, extraWait time.Duration) map[InformerName]error {
 	resultChan := make(chan syncInformerResult)
 	errors := make(map[InformerName]error, len(informers))
+
+	if len(informers) == 0 {
```

Comment:
nit: no need to allocate with `make` without informers, it can be first check and return `nil`

Reply:
https://github.com/DataDog/datadog-agent/pull/34773/commits/570afa194de97c425185ad05452549ea301398c2